public/scripts/contollers/articleController.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+(function(module) {
+  const articleController = {};
+  articleController.index = () => {
+    Article.fetchAll(articleView.initIndexPage);
```

Comment:
Doesn't this look like a good place to handle a call moved from index.html? Maybe a check of the ETag prior to loading data makes sense here or within a function being called here?